### PR TITLE
Point weight

### PR DESCRIPTION
**Pull request by shun-iwasawa**
image
_Left: Every correnspondence points have the same weight. The shape is evenly deformed.
Right: Increase the weights at the four corners to 3 and leave the weight of the center shape to 1. The mesh is stretched to the left and right, creating an uneven distortion like that of a cylinder._

This PR will add a new parameter, `weight` to each correspondence point.
The higher the value of the `weight`, the more the mesh vertices are attracted.
In other words, where the `weight` parameters are large, the mesh is denser and the picture looks squeezed.
This allows you to represent a shape with gradually tilting surfaces, such as a cylinder, for example.

The `weight` value of the selected points can be adjusted in the tool options panel for the Correspondence tool.

### CODE REVIEW

**Comment by RodneyBaker** [comment]
Very nice addition @shun-iwasawa 

pointweight

This just a quick test of changing upper and lower left corner point weights to 5 half way through a sequence and then back to 1 again.
